tidy3d/components/simulation.py

```diff
@@ -8,7 +8,7 @@
 import pydantic
 import numpy as np
 import xarray as xr
-import matplotlib.pylab as plt
+import matplotlib.pyplot as plt
```

Comment:
does this have any practical effects in our plotting?

Reply:
No, `plt` is only used in `plt.colorbar(mappable, cax=cax, label=r"$\epsilon_r$")` in this file, and `colorbar` works the same way in both interfaces.